build.gradle

```diff
@@ -127,10 +127,10 @@ forbiddenPatterns {
 // TODO: fix license. skip dependency license checks
 dependencyLicenses.enabled = false
 
-// We don't need to following ES naming conventions.
-// see https://github.com/elastic/elasticsearch/blob/5ca6f312058ec8c7fbac072c4648246e7dfb4957/buildSrc/src/main/java/org/elasticsearch/gradle/precommit/NamingConventionsTask.java
-// enable namingConvention check will cause errors like:  "Classes ending with [Tests] must subclass [LuceneTestCase]"
-namingConventions.enabled = false
+// We don't need to following ES testing naming conventions.
```

Comment:
NP: Should this say "We don't need to ```follow``` ES testing naming conventions"
or "We don't need ```the``` following ES testing naming conventions"?

Reply:
Right, fixing it. Thanks!

Reply:
Done!